CONTRIBUTING.md

```diff
@@ -4,20 +4,42 @@
     1. Merge an update to main with a new release number in the .json file. 
     2. Have someone from SecOps change the version in the Splunk SOAR interface.
 
+## Setup
+
+Python version should be set to 3.9 to match Splunk SOAR.
+
+```shell
+pyenv install 3.9.0
+pyenv local 3.9.0
+```
+
+You can verify the above with `pyenv local`:
+```
+$ pyenv local
+3.9.0 (set by /home/delaport/src/secops-splunk-null-router/.python-version)
+```
```

Comment:
I prefer verifying python directly in case the user is relying on global instead, or in case pyenv is broken in any way:

````suggestion
You can verify the above with `python --version`:
```
$ python --version
Python 3.9.0
```
````